playground/infrastructure/helper.py

```diff
@@ -0,0 +1,76 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import List
+
+from api.v1.api_pb2 import Sdk, SDK_UNSPECIFIED, Status, STATUS_UNSPECIFIED
+
+
+class Example:
+    """ Class which contains all information about beam example
+    """
+    name = ""
+    sdk = Sdk
+    filepath = ""
+    code = ""
+    output = ""
+    status = Status
```

Comment:
This way you declare class variables, it seems that it is not what you want to do (well, it's Python, not Java😁). Just remove it, the code in the `__init__` function should be enough.

Reply:
It should be changed but don't use __init__ - it is better to use dataclass (https://towardsdatascience.com/9-reasons-why-you-should-start-using-python-dataclasses-98271adadc66)
![image](https://user-images.githubusercontent.com/75420442/142196422-9499eb98-530b-4c07-823c-2b0729997900.png)

Reply:
This approach works too :)

Reply:
Changed.